### ISSUE TITLE
Adding pip support to environments

### ISSUE THREAD

**Issue by costrouc** [Quansight/qhub-ops#50]
Depends on https://github.com/Quansight/conda-store/issues/1

**Comment by scopatz** [comment]
This is proabably a bad idea in general, but I can see how it is useful.

**Comment by costrouc** [comment]
I agree... I would really like to encourage people to only have conda packages in their environments. But in practice what I've seen from several of our internal projects there always end up being pip dependencies that users want to add. Either way I fixed it and it supports the normal conda environment spec with pip packages.
